### PR TITLE
New addresses component

### PR DESCRIPTION
**Pull request by Tommytrg**
Apply new design of the addresses component and the necessary components to build it.

### CODE REVIEW

File: src/components/AddressInformation.vue [hunk 1]
```diff
@@ -89,6 +91,14 @@ export default {
       copied: false,
     }
   },
+  computed: {
+    blindPkh() {
+      const start = this.pkh.slice(0, 20)
+      const end = this.pkh.slice(29)
```
Comment (aesedepece) [inline src/components/AddressInformation.vue:97]:
```suggestion
      const end = this.pkh.slice(-16)
```

File: src/components/AddressInformation.vue [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+    font-style: italic;
+    line-height: 150%;
+
+    .link {
+      font-weight: 500;
+      color: $link-grey;
+      text-decoration: underline;
+    }
+  }
+}
```
Comment (aesedepece) [inline src/components/AddressInformation.vue:220]: Make sure the margins of this element match de design :wink:

File: src/components/Addresses.vue [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/styles/_colors.scss';
+
+.card {
+  border: 1px solid $grey-1;
+  border-radius: 4px;
+  /* height: 370px; */
```
Comment (gabaldon) [inline src/components/Addresses.vue:80]: Add  ```overflow: hidden;``` to avoid border-radius clip the content.

Comment (aesedepece) [reply src/components/Addresses.vue:80]: I was about to comment exactly the same hahaha

**Comment by aesedepece** [comment]
Please use `Roboto Mono` for addresses. We could even make it a default for `pre` elements.

**Comment by aesedepece** [comment]
The padding of the used / not used label look slightly off to me:

image

I think the card could also use some more bottom padding, but I haven't measured if it's the same as in figma, so it could be also wrong in the design.

**Comment by aesedepece** [comment]
Just a reminder that we need to come with a better empty state:
image

**Comment by Tommytrg** [comment]
> Just a reminder that we need to come with a better empty state (specially `#-1` doesn't make much sense hahaha):
> 
> image

I didn't think in the empty state assuming the wallet always generates at least one address. If it isn't the case, I will add an empty state.

**Comment by aesedepece** [comment]
> I didn't think in the empty state assuming the wallet always generates at least one address. If it isn't the case, I will add an empty state.

Yeah let's have an empty state until we make sure the assumption from the wallet becomes true. Sorry if a few days ago I provided confusing feedback about this :wink:

**Comment by aesedepece** [comment]
When you have two rows of addresses, the vertical space between the rows is excessive. Funnily enough, when a third row is added, everything looks just fine.

**Comment by aesedepece** [comment]
I guess it is a known limitation, but this component will not show more than 25 addresses at the same time (probably the issue is not in the component itself but in `getAddresses`).

**Comment by mariocao** [comment]
Yesterday I checked the code for `getAddresses` and it seemed to the same as `getTransactions`. I will double-check it again. :)

**Pull request merged**